.changelog/2537.trivial.md

```diff
@@ -1 +0,0 @@
-runtime: Extract common sealing code.
```

Comment:
Was this forgotten? How did that happen?

Reply:
So, I ran `make changelog` on the `master` branch before #2537 was merged and submitted the Change Log for 20.0 PR (#2540).
Since #2537 was merged before the Change Log PR and I saw that the change was trivial, i.e. it didn't impact the Change Log, I just rebased my PR on top of it. But this then didn't delete the `2537.trivial.md` fragment :slightly_smiling_face:.